{"name": "test_001", "status": "failed", "statusDetails": {"message": "AssertionError: assert 0", "trace": "self = <scripts.test_case.TestCase object at 0x000001C8074AF730>\n\n    def test_001(self):\n>       assert 0\nE       assert 0\n\nscripts\\test_case.py:3: AssertionError"}, "start": 1748149426357, "stop": 1748149426357, "uuid": "8abe921b-257a-48e9-af77-346dde98e329", "historyId": "b52620e2fa88e5ec1efe61251c01560f", "testCaseId": "b52620e2fa88e5ec1efe61251c01560f", "fullName": "scripts.test_case.TestCase#test_001", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test_case"}, {"name": "subSuite", "value": "TestCase"}, {"name": "host", "value": "LAPTOP-47HNDVHS"}, {"name": "thread", "value": "30148-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test_case"}]}